### PR TITLE
Added a new service to interact with the BC component

### PR DESCRIPTION
**Pull request by CamiloNicolasMartinez**

### CODE REVIEW

**Review by agustico13 — COMMENTED** [review]
Good

**Comment by ismaelpuricelli** [comment]
Very good!

**Review by ismaelpuricelli — APPROVED** [review]
Very good!

**Review by agustico13 — APPROVED** [review]

**Review by juan17rodriguez — APPROVED** [review]
Great job!

File: src/server/services/blockchain.ts [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+                'Content-Type': 'application/json'
+            },
+            json: true,
+        }).then((res) => {
+            logger.info(`Obtained the transactions of the user ${userId}`)
+            return res
+        }).catch((err) => {
+            logger.error(`Failed to obtain the transactions of the user  ${userId} - ${JSON.stringify(err)}`)
+        })
+        logger.info(`TRANSACTIONS => ${transactions} ${JSON.stringify(transactions)}`)
```
Comment (CamiloNicolasMartinez) [inline src/server/services/blockchain.ts]: Remove this

**Pull request merged**